### PR TITLE
This commit adds the gatsby-starter-minimal-blog plugin to the website.

### PR DESCRIPTION
**Pull request by traytorous**
It changes the /about route to /purpose and all links to /about has been changed to point to /purpose.
This is because the plugin by default configues the /about route to point to a page that is automatically generated

### CODE REVIEW

**Comment by bryanmontalvan** [comment]
There seems to be some NPM errors, @traytorous does the website compile and run fine locally?

**Comment by traytorous** [comment]
@bryanmontalvan there are some dependency issues. It's due to the fact that some of libraries for the blog are older than the ones from the official site. Marcel wanted me to create a pull request just so that he could look at it and so that more people could hop on it. The website runs locally but you have to invoke the --force command. The other tests are failing from what i've seen due to the fact that I had to change the  routes of the site. Check out my comments here to get more info https://github.com/operate-first/operate-first.github.io/issues/80

**Comment by traytorous** [comment]
fixed it. Had to downgrade react by a level

**Comment by bryanmontalvan** [comment]
@traytorous The hero container's height seems to be funky, although I'm unsure how this came to be. Does it look fine/normal on your local environment? (The height on mine has shrunk)

image

File: package.json [hunk 1]
```diff
@@ -17,16 +17,18 @@
...
-    "gatsby-plugin-google-analytics": "^4.14.0",
-    "gatsby-plugin-manifest": "^4.14.0",
+    "@lekoarts/gatsby-theme-minimal-blog": "^4.1.7",
+    "@mantine/core": "^4.2.10",
+    "@mantine/hooks": "^4.2.10",
+    "gatsby": "^4.16.0",
+    "gatsby-plugin-google-analytics": "^4.16.0",
+    "gatsby-plugin-manifest": "^4.16.0",
     "gatsby-plugin-mantine": "^4.0.0",
-    "react": "^18.1.0",
```
Comment (durandom) [inline package.json:26]: do we really need to downgrade react? 
the original lekoarts engine also depends on 18 https://github.com/LekoArts/gatsby-starter-minimal-blog/blob/master/package.json

**Comment by durandom** [comment]
TBH I'm actually not so sure anymore if adding `@lekoarts/gatsby-theme-minimal-blog` to package.json is the way to go. It seems to add CSS to our site.

The goal is to have https://www.operate-first.cloud/blog and https://www.operate-first.cloud/blog/kube-con-valencia-eu-2022-trip-report render with the same layout as the current site, which doesn't work out of the box.

How about we just use gatsby-theme-minimal-blog as an inspiration to add blogging capabilities to our site? I.e. inspect https://github.com/LekoArts/gatsby-themes/tree/main/themes/gatsby-theme-minimal-blog-core and re-implement in our site.

Or, as another experiment, depend on https://github.com/LekoArts/gatsby-themes/tree/main/themes/gatsby-theme-minimal-blog-core instead of the themed version and see how far we get from there.

File: package.json (hunk 1 above)
Comment (traytorous) [reply package.json:26]: You're right however to merge  our website together it needed a dependency called "gatsyby-plugin-mdx"
which requires react version 17 or 16.  If you try to upgrade react to version 18 you will get the error shown below. 
PeerDependency-gatsby-plugin

**Pull request closed without merging**